e2e-tests/commands/insurance/create-an-application.js

```diff
@@ -0,0 +1,31 @@
+import api from '../api';
+import mockApplication from '../../fixtures/application';
+
+const { ELIGIBILITY: mockEligibilityAnswers } = mockApplication;
+
+/**
+ * createAnApplication
+ * Create an application directly via the API
+ * 1) Get a country
+ * 2) Create a buyer with country ID
+ * 3) Generate mock application with owner and buyer relationships
+ * 4) Create applications
+ * @param {String} Account ID for the application owner
+ * @param {Number} Amount of applications to create
```

Comment:
extra param to remove?

Reply:
Discussed, will merge and address this in another PR so that we can get quicker tests in all other PRs.